### ISSUE TITLE
Toolbox with available resources

### ISSUE THREAD

**Issue by kamil-mrzyglod** [msshli/arm-visualizer#9]
Hi,

are you planning to implement something like a toolbox with available resources, which could be dragged and dropped in the editor?

**Comment by shenglol** [comment]
Yes. Toolbox is one of the highest priority feature to be implemented. But I may no be able to actively maintain this repo until next month.

**Comment by kamil-mrzyglod** [comment]
Cool, if you need any help, please ping me.
